library/src/test/java/com/github/pwittchen/reactivenetwork/library/PreconditionsTest.java

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright (C) 2016 Piotr Wittchen
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.github.pwittchen.reactivenetwork.library;
+
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.robolectric.RobolectricTestRunner;
+import org.robolectric.annotation.Config;
+
+import static com.google.common.truth.Truth.assertThat;
+
+@RunWith(RobolectricTestRunner.class) @Config(constants = BuildConfig.class)
+public class PreconditionsTest {
+
+  @Test @Config(sdk = 21) public void shouldBeAtLeastAndroidLollipop() {
```

Comment:
Oh, really? That surprises me. Are you sure because I can remember using `sdk = 19` on a test class a couple of years ago... Possibly with Robolectric 2.4 or something like that...